proto/ddl_service.proto

```diff
@@ -168,6 +168,23 @@ message AlterRelationNameResponse {
   uint64 version = 2;
 }
 
+message AlterOwnerRequest {
+  oneof entity {
```

Comment:
```suggestion
  oneof object {
```
Since we have already used  keyword `object` for them. Better to keep consistency. https://github.com/risingwavelabs/risingwave/blob/e3c8649ec4b3c5adb3e37d1dd82e8e421be1f8ee/proto/user.proto#L55